utils/debug/Debug.js

```diff
@@ -4,45 +4,21 @@ import Coordinates from '../../src/Core/Geographic/Coordinates';
 import OBBHelper from './OBBHelper';
```

Comment:
in Debug.js :
   75:6  error  Missing semicolon  semi
  151:6  error  Missing semicolon  semi
  210:6  error  Missing semicolon  semi